### PR TITLE
Add tts view and model, remove hardcoded date

### PR DESCRIPTION
**Pull request by TomSavas**
@ignas-gustainis 
Do you know how to patch the tts ViewSet so it returns a single object of an array?
Right now it returns this:
```
[
   {
      startTime: "2017-08-16T09:00:00Z",
      endTime: "2017-08-26T09:00:00Z"
   }
]
```
I'd like it to be this:
```
{
   startTime: "2017-08-16T09:00:00Z",
   endTime: "2017-08-26T09:00:00Z"
}
```
As far as I understand we are using queryset in the ViewSet which in it self is an array of sort.
We could write tts view as a simple view (as we did before)?

### CODE REVIEW

**Comment by ignas-gustainis** [comment]
Definitely, just use a simple view - ViewSet is designed to return specific sets of objects - to fetch the object in the view, you can probably just use `TimeTillSession.objects.last()`

File: nma/settings.py [hunk 1]
```diff
@@ -124,8 +124,9 @@
 STATIC_ROOT = os.path.join(BASE_DIR, 'static')
 
 REST_FRAMEWORK = {
-    # Use Django's standard `django.contrib.auth` permissions,
-    # or allow read-only access for unauthenticated users.
+    'DEFAULT_RENDERER_CLASSES': (
```
Comment (ignas-gustainis) [inline nma/settings.py]: This change is not related - isn't this change already comitted?
You can probably fix this by running `git rebase origin/master`

**Comment by TomSavas** [comment]
@ignas-gustainis 
When I tried the old view it returns the date in some other format:
`2017-08-16 09:00:00+00:00`
But it does return the date in T-Z format if I'm using the rest framework...
So I hacked a solution: redirecting from /tts to a site with ID, so the format is T-Z and we get rid of the square brackets.
Do you approve this way? :smiley:

**Comment by ignas-gustainis** [comment]
It likely won't work when fetching the API through mobile (though I am not 100% sure) - since you are using RedirectView, it first sends a 302 HTTP redirect header and the URL to redirect to - the redirection itself is left to the user. Your browser might follow the redirection, however, the mobile client that will fetch the API will likely not. Try to avoid using redirections in APIs at all costs.

**Comment by ignas-gustainis** [comment]
Why not just format the time format by yourself in the simple view - `time.strftime("%Y-%m-%dT%H:%M:%SZ", TIME_VARIABLE_HERE)`

**Comment by TomSavas** [comment]
What do you think? Will this do?

File: nma/views.py [hunk 2]
```diff
@@ -29,6 +31,17 @@ class NewsViewSet(viewsets.ModelViewSet):
...
+    stringStartTime = str(TimeTillSession.objects.get(id=1).startTime)[:-6]
+    structStartTime = time.strptime(stringStartTime, "%Y-%m-%d %H:%M:%S")
+    tzStartTime = time.strftime("%Y-%m-%dT%H:%M:%SZ", structStartTime)
+
+    stringEndTime = str(TimeTillSession.objects.get(id=1).endTime)[:-6]
+    structEndTime = time.strptime(stringEndTime, "%Y-%m-%d %H:%M:%S")
+    tzEndTime = time.strftime("%Y-%m-%dT%H:%M:%SZ", structEndTime)
+
+    responseDict = {'startTime': str(tzStartTime), 'endTime': str(tzEndTime)}
+    response = json.loads(json.dumps(responseDict))
```
Comment (ignas-gustainis) [inline nma/views.py]: This line is unneeded - just return JsonResponse(responseDict) (currently, you are converting the dict to a json string and then back into dict)

**Comment by ignas-gustainis** [comment]
Other than the comment I posted, definitely, looks good to me :+1:, good job :)
